dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -1612,11 +1615,19 @@ public ManifestInfo getManifestInfo() {
 	}
 
     public void setVariantId(final String variantId) {
-		map.put(VARIANT_ID, variantId);
+		if (UtilMethods.isSet(variantId)) {
+			this.variantId = variantId;
+		} else {
+			this.variantId = VariantAPI.DEFAULT_VARIANT.name();
+		}
```

Comment:
this.variantId = UtilMethods.isSet(variantId) ? variantId : VariantAPI.DEFAULT_VARIANT.name();

Reply:
done https://github.com/dotCMS/core/pull/25035/commits/4a7b5555a6623425addbb8ea468a2a3201b4d144#diff-1367236d40a612b923ececfa521053143f026f60f035c9f9963aae4efdfaf034R1618